pages/linux/entr.md

```diff
@@ -0,0 +1,23 @@
+# entr
+
+> Run arbitrary commands when files change.
+
+- Rebuild with `make` if any files in any subfolders change:
+
+`ag -l | entr make`
```

Comment:
Perhaps we could enclose `ag -l` and `make` in quotes? This denotes that they can be changed by the reader.

The same goes for the examples further down.

Reply:
Oops! I meant the token syntax :P

Reply:
Aha, that makes more sense. I've added a commit (5d0376) that puts all non-flag commands in `{{` and `}}` brackets. Is that what you meant?